### ISSUE TITLE
Allow mounts to be used during server installation

### ISSUE THREAD

**Issue by leonardopessoa** [pterodactyl/panel#4520]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

Similar to the items below, mounts would be a great way to cache big games like CSGO if they were available at the server installation time. Moreover, you could create fully modded versions to work like a master (for e.g. in CSGO: pre-installed SourceMod, additional plugins, branding, etc) that new servers would copy from.

https://github.com/pterodactyl/panel/issues/119
https://github.com/pterodactyl/panel/issues/3556

Please feel free to chime in if there is another paradigm/feature currently available that meets this idea.

### Describe the solution you'd like.

- Enable mount at server installation
- Installation scripts could leverage the mount to leverage pre-built (customized/configured) game servers

**Comment by devnote-dev** [comment]
This has been addressed in #4034 but that PR needs updating.

**Comment by leonardopessoa** [comment]
> This has been addressed in #4034 but that PR needs updating.

Right, but that one is closed. Should it be reopened then and close this one for further discussion?

**Comment by devnote-dev** [comment]
> Right, but that one is closed. Should it be reopened then and close this one for further discussion?

It hasn't been closed, but it is outdated by a few versions. This issue can be kept open for tracking purposes if that one is closed or superseded.

**Comment by alexevladgabriel** [comment]
Looking to implement this feature soon again and update it to latest version.

**Comment by Juli199696** [comment]
Any update on this feature? @alexevladgabriel

**Comment by ItsCEED** [comment]
bump @alexevladgabriel

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
